site/networking.xml

```diff
@@ -0,0 +1,697 @@
+          algorithm</a> is primarily useful for reducing latency but
+          can also improve
+          throughput. <code>kernel.inet_default_connect_options</code>
+          and <code>kernel.inet_default_listen_options</code> must
+          include <code>{nodelay, true}</code> to disable Nagle's
+          algorithm for inter-node connections.  When configuring
+          sockets that serve client connections,
+          <code>rabbit.tcp_listen_options</code> must include the same
+          option. This is the default.
+
+          The following example demonstrates that:
+
+<pre class="code">
+[
+  {kernel, [
+    {inet_default_connect_options, [{nodelay, true}]},
+    {inet_default_listen_options,  [{nodelay, true}]}
+  ]},
+  {rabbit, [
+    {tcp_listen_options, [
+                          {nodelay, true}
+                          ]}
+  ]}
+].
+</pre>
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="tuning-for-large-number-of-connections-async-thread-pool">
+        <doc:heading>Erlang VM I/O Thread Pool Tuning</doc:heading>
+        <p>
+          Adequate Erlang VM I/O thread pool size is also important when tuning for a large number of
+          concurrent connections. See the section above.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="tuning-for-large-number-of-connections-connection-backlog">
+        <doc:heading>Connection Backlog</doc:heading>
+        <p>
+          With a low number of clients, new connection rate is very unevenly distributed
+          but is also small enough to not make much difference. When the number reaches tens of thousands
+          or more, it is important to make sure that the server can accept inbound connections.
+          Unaccepted TCP connections are put into a queue with bounded length. This length has to be
+          sufficient to account for peak load hours and possible spikes (e. g. mass client
+          disconnects due to a network interruption). This is configured using the <code>rabbit.tcp_listen_options.backlog</code>
+          option:
+
+<pre class="code">
+[
+  {rabbit, [
+    {tcp_listen_options, [{backlog, 4096}]}
+  ]}
+].
+</pre>
+
+           Default value is 128. When pending connection queue length grows beyond this value,
+           connections will be rejected by the operating system. See also <code>net.core.somaxconn</code>
+           in the kernel tuning section.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+    <doc:section name="os-tuning">
+      <doc:heading>OS Level Tuning</doc:heading>
+
+      <doc:subsection name="os-tuning-intro">
+        <p>
+          Operating system settings can affect operation of RabbitMQ.
+          Some are directly related to networking (e.g. TCP settings), others
+          affect TCP sockets as well as other things (e.g. open file handles limit).
+
+          Understanding these limits is important, as they may changing depending on
+          the workload.
```

Comment:
There is a mistake in the sentence "they may changing depending the workload". Did you mean "they may change, depending (...)"?